index.html

```diff
@@ -40,5 +44,202 @@ <h1>I´m Josué Glad to see you!</h1>
+        <button type="button" class="javaScript">javaScript</button>
+    </form>
+    <div>
+        <input type="button" name="boton1" value="See Project">
+    </div>
+    </header>
+    </section>
+
+
+<!--Fifth part-->
+
+
+<section class="container-5">
+    <header class="image-header">
+    <h2>
+    <img src="images/Snapshoot4.png" alt="Snapshoot4" class="img">
+    </h2>
+    <div class="multi">
+        <h6>Multi-Post Stories</h6>
+    </div>
+    <div class="list-4">
+        <ul>
+            <li class="canopy">CANOPY</li>
+            <li class="back-end">Back end dev</li>
+            <li class="num">2015</li>
+        </ul>
+    </div>
+    <div class="text">
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <form class="buttom">
+        <button type="button" class="html">html</button>
+        <button type="button" class="css">css</button>
+        <button type="button" class="javaScript">javaScript</button>
+    </form>
+    <div>
+        <input type="button" name="boton1" value="See Project">
+    </div>
+    </header>
+    </section>
+
+
+<!--sixth part-->
+
+<section class="container-6">
+    <div class="title">
+        <h1 class="about">About Myself</h1>
+    </div>
+    <div class="description-test">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <div class="lets-connect">
+        LET´S CONNECT
+    </div>
+    <div class="icons">
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fa-brands fa-google"></i>
+        <i class="fa-brands fa-facebook"></i>
+        <i class="fa-brands fa-whatsapp"></i>
+        <i class="fa-brands fa-github"></i>
+        <i class="fa-brands fa-instagram"></i>
+        <i class="fa-brands fa-linkedin-in"></i>
+    </div>
+    <div>
+        <input type="button" name="boton1" value="Get my resume">
+    </div>
+</section>
+
+<!--Seventh part-->
+
+<section class="conatiner-7">
+    <div class="languajes">
+     <h2>Languajes
+    <i class="fa-solid fa-angle-down"></i>
+    </h2>
+    </div>
+    <div class="two-list">
+        <ul class="works">
+            <li class="java"><img src="images/Ellipse1.png" alt="js" class="js">JavaScript</li>
+            <li class="ht"><img src="images/Ellipse2.png" alt="Html" class="ht">HTML</li>
+            <li class="cs"><img src="images/Ellipse3.png" alt="css" class="cs">CSS</li>
+        </ul>
+    </div>
+    <div class="frameworks">
+        Frameworks
+        <i class="fa-solid fa-angle-right"></i>
+    </div>
+    <div>
+        <hr>
+    </div>
+    <div class="skills">
+        Skills
+        <i class="fa-solid fa-angle-right"></i>
+        <div>
+            <hr>
+        </div>
+    </div>
+</section>
+<section class="form-section">
```

Comment:
Your elements should be laid out using flexbox. Please fix this.

<img width="256" alt="Screen Shot 2022-06-08 at 13 00 34" src="https://user-images.githubusercontent.com/82426682/172685543-48577332-0c04-4593-b876-64bd0b2fb230.png">